types/index.d.ts

```diff
@@ -11,6 +11,12 @@ export const enum Level {
 export interface ChalkOptions {
 	enabled?: boolean;
 	level?: Level;
+	wrapper?: Wrapper;
+}
+
+export interface Wrapper {
```

Comment:
Can you define these inline. Doesn't really make sense to expose a separate `interface` for it.